src/components/td/td-styles.jsx

```diff
@@ -20,12 +20,61 @@ export default theme => {
       minWidth: 20,
       paddingTop: 8,
       paddingBottom: 8,
+      display: 'block',
+
+      '&::before': {
+        display: 'block',
+        content: 'attr(data-title)',
+      },
 
       [mixins.media('md')]: {
         fontSize: 14,
+        display: 'inline-block',
         minWidth: 40,
         paddingTop: 10,
         paddingBottom: 10,
+        '&::before': {
+          content: 'none',
+        },
+      },
+
+      '&.flexOrder': {
+        order: props => props.flexOrder,
+
+        [mixins.media('md')]: {
+          order: () => 1,
+        },
+      },
+
+      '&.flexBasis': {
+        flexBasis: props =>
+          props.flexBasisMobile
+            ? `${props.flexBasisMobile}${typeof props.flexBasisMobile === 'number' ? '%' : ''}`
+            : `${props.flexBasisDesktop}${typeof props.flexBasisDesktop === 'number' ? '%' : ''}`,
+
+        [mixins.media('md')]: {
+          flexBasis: props => `${props.flexBasisDesktop}${typeof props.flexBasisDesktop === 'number' ? '%' : ''}`,
+        },
+      },
+
+      '&.width': {
+        width: props => `${props.width}${typeof props.width === 'number' ? '%' : ''}`,
+      },
+
+      '&.align': {
+        textAlign: props => (props.alignMobile ? props.alignMobile : props.alignDesktop || props.align),
+
+        [mixins.media('md')]: {
+          textAlign: props => props.alignDesktop || props.align,
```

Comment:
This will break in IE11 due to a bug with dynamic props and nested jss rules mantioned here: https://github.com/cssinjs/jss/issues/623

Reply:
We'll fix it, we see the problem. Toggle on classname instead of nested dynamic props